ledfx/effects/random.py

```diff
@@ -0,0 +1,91 @@
+import random
+import timeit
+
+import numpy as np
+import voluptuous as vol
+
+from ledfx.color import parse_color, validate_color
+from ledfx.effects.temporal import TemporalEffect
+
+
+class RandomEffect(TemporalEffect):
+    """
+    Randomly activates sections
+    """
+
+    NAME = "Random"
```

Comment:
I would recommend a name change to something like lightning, more descriptive of intent. Random is going to be interpreted to widely.

Changing the name is as simple as changing the file name, class name and this NAME, its easier than it sounds.

I still would recommend looking into implementing a decay on the "strike" which is as simple as a float 1->0 decay multiplier of the color array. It will look a lot more organic.